app/data/jsonSchemaForm/projectEmissionIntensitySchema.ts

```diff
@@ -2,62 +2,76 @@ export const emissionIntensityReportSchema = {
   $schema: "http://json-schema.org/draft-07/schema",
   type: "object",
   title: "Emissions Intensity Report",
-  required: [
-    "measurementPeriodStartDate",
-    "measurementPeriodEndDate",
-    "emissionFunctionalUnit",
-    "baselineEmissionIntensity",
-    "targetEmissionIntensity",
-  ],
   properties: {
-    measurementPeriodStartDate: {
-      title: "TEIMP Start Date",
-      type: "string",
+    teimpReporting: {
+      $schema: "http://json-schema.org/draft-07/schema",
```

Comment:
the JSON schema object only has one declartion of the spec version at the top level, this is unnecessary
```suggestion

```